### ISSUE TITLE
about:accounts - TOS/PP text fades in, password length text fades out.

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa-content-server#2241]
@johngruen - this may be due to the SASS refactor.

![about-accounts-startup-jank](https://cloud.githubusercontent.com/assets/848085/6847321/904739ea-d3bd-11e4-9b1e-0f33821c552f.gif)

STR
1. Create a new Firefox profile.
1. Update the profile's config to point to latest.dev.lcip.org
1. Load up `about:accounts`
1. Click "Get Started"
1. Yuck.

Here is my user.js with the values already set up:
```js
/* copy the pref group below into a user.js file created in profile dir. e.g.
 * Mac: /Users/<user>/Library/Application\ Support/Firefox/Profiles/<obfuscate_chars>.default/user.js
 * Win: C:/Users/Username/Appdata/Roaming/Mozilla/Firefox/Profiles/<obfuscate_chars>.default/user.js
 * Lnx: ~/.mozilla/firefox/<obfuscate_chars>.default/user.js
 * Restart Firefox
*/

// local
user_pref("general.warnOnAboutConfig", false);
user_pref("services.sync.log.appender.file.logOnSuccess", true);
user_pref("identity.fxaccounts.allowHttp", true);
user_pref("identity.fxaccounts.auth.uri", "https://latest.dev.lcip.org/auth/v1");
user_pref("identity.fxaccounts.remote.force_auth.uri", "https://latest.dev.lcip.org/force_auth?service=sync&context=fx_desktop_v1");
user_pref("identity.fxaccounts.remote.signin.uri", "https://latest.dev.lcip.org/signin?service=sync&context=fx_desktop_v1");
user_pref("identity.fxaccounts.remote.signup.uri", "https://latest.dev.lcip.org/signup?service=sync&context=fx_desktop_v1");
user_pref("identity.fxaccounts.settings.uri", "https://latest.dev.lcip.org/settings");
user_pref("services.sync.tokenServerURI", "https://latest.dev.lcip.org/syncserver/token/1.0/sync/1.5");
user_pref("services.sync.debug.ignoreCachedAuthCredentials", true);
user_pref("app.update.auto", false);
user_pref("app.update.enabled", false);
user_pref("app.update.silent", false);
user_pref("app.update.staging.enabled", false);
```

**Comment by johngruen** [comment]
@shane-tomlinson i'm afraid this is happening in prod as well, so it's not the Sass refactor, it's just the way things are. Still, I'll fix it.

**Comment by shane-tomlinson** [comment]
> @shane-tomlinson i'm afraid this is happening in prod as well, so it's not the Sass refactor, it's just the way things are. Still, I'll fix it. 

Ahha. I only tried once on prod, didn't see it, and assumed a sass refactor regression. I just tried prod again, and yup, the problem is there too.

**Comment by johngruen** [comment]
@shane-tomlinson this was fixed on #2248
